doc/content/guides/semtech-interop/_index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Interoperability with Semtech Join Server"
+description: ""
+weight: 40
+---
+
+## Interoperability Repository
+
+In order to enable interoperability in the stack, create a interoperability configuration repository(e.g. a directory) with `config.yaml` at the root.
+
+For example:
+```bash
+$ mkdir interopconf
+$ touch interopconf/config.yaml
+```
+
+`config.yaml` contains various interoperablity configuration options.
+The option we're interested at is `join-servers`.
+`join-servers` is a list of Join Server interoperability configurations.
+Each Join Server interoperability configuration allows 2 options to be specified:
+- `file` - relative path to a file containing Join Server configiration.
+- `join-euis` - list of Join EUI prefixes the Join Server should handle.
+
+In case Join EUI prefixes overlap, the most specific prefix takes precedence.
+
+For example:
+`interopconf/config.yaml` content:
+```yaml
+join-servers:
+  - file: ./semtech/js.yaml
+    join-euis:
+      - 000e000000000000/16
+      - 000f000000000000/16
+
+  - file: ./ttn/js.yaml
+    join-euis:
+      - 70b3d57ed0000000/40
+```
+
+A configuration like this would make Join-Requests for JoinEUIs starting with `000E` or `000F` to be handled by Join Server defined at `semtech/js.yaml` in interoperability mode and all other ones to be handled by Join Server defined at `tti/js.yaml`.
+The prefix specified for the Semtech Join Server should match the JoinEUI of the devices you would like to use Semtech Join Server.
+NOTE: All paths are relative to the file they're defined in, that is `semtech/js.yaml` defined in `interopconf/config.yaml` is expected to be located at `interopconf/semtech/js.yaml`.
+NOTE(2): Interoperability Join Server requests are only performed if either:
+a. Cluster-local Join Server is not available(e.g. it's disabled in the stack or it's down)
+b. Cluster-local Join Server is available, but does not contain the information about a device indentified by particular JoinEUI:DevEUI pair.
+NOTE(3): `0000000000000000/0` prefix matches all JoinEUIs, while being the least specific, hence it always matches last. Use this prefix if you wish to define a "fallback" Join Server interoperability configuration.
+
+## Join Server Configuration
+
+The Join Server configuration provides means to configure how the components interact with the Join Server.
+The configuration supports multiple options, however the options required for Semtech Join Server in particular, are:
```

Comment:
Also here empty lines for breaking paragraphs.

Reply:
Do not use single line breaks basically. If things belong in the same paragraph, just use one line and enable word wrap in your editor to make your life more bearable

Reply:
Also is this a good way to show yaml? How about a yaml example with the description in the values?

Reply:
Yaml does not support comments, so how do you suggest to add the description?
Do you mean:
```yaml
some-option: You value could be here
```
?

Reply:
Hmm yeah something like [this](http://github.com/thethingsnetwork/lorawan-stack/tree/master/docker-compose.yml)

Reply:
Right, `yaml` does support comments. Must have confused it with some other `*ml`

Reply:
JSON doesn't support it